client/src/store/new-request-module.ts

```diff
@@ -2024,11 +2054,10 @@ export class NewRequestModule extends VuexModule {
       // if there is no previous request_action message then we just preserve whatever text there is and append msg
       data['additionalInfo'] += ` \n\n ${msg}`
       return data
-    } catch (error) {
-      console.error('addRequestActionComment() =', error) // eslint-disable-line no-console
-      await errorModule.setAppError(
-        { id: 'add-request-action-error', error: 'An error occurred when building the name request' }
-      )
+    } catch (err) {
+      const msg = await handleApiError(err, 'Could not add request action comment')
+      console.error('addRequestActionComment() =', msg) // eslint-disable-line no-console
+      await errorModule.setAppError({ id: 'add-request-action-error', error: msg } as ErrorI)
```

Comment:
FYI, this is the other way errors are handled: add error to error array, which automatically displays the error modal.
(The only thing is, the error modal no longer displays error details. But we can clean that up later. Or perhaps repurpose this to log error differently, I dunno.)